### ISSUE TITLE
Most tests fail with a "Timeout" result

### ISSUE THREAD

**Issue by ZaLiTHkA** [GoogleChromeLabs/container-query-polyfill#41]
considering that we're getting closer to "Container Queries" becoming stable, I thought it would be a good plan to start making use of them in the meantime, initially powered by the use of a polyfill...

unfortunately though, it seems the "tests" included in this repository don't really work properly.

the environment I ran this in is as follows:

* Windows 10 21H2
* Google Chrome 101.0.4951.67 (with `enable-container-queries` turned on)
* NodeJS 16.15.0

when I view the `http://localhost:8081/tests` URL in my browser, I see the following output:

![image](https://user-images.githubusercontent.com/2149312/169016156-75e1ff94-6cf1-4c8a-b70b-726bacaa775c.png)

---

note: I have not tested the polyfill itself in a _custom_ project, but seeing the output of the built-in tests, I'm not so sure it's worth going down that rabbit hole..

to ask the same thing as #40: is this project dead...? otherwise, is there another polyfill somewhere that we should be using instead?

**Comment by ZaLiTHkA** [comment]
note: I did also find [`CQFill`](https://github.com/jsxtools/cqfill), as used in the ["Container Queries with Polyfill and Grid"](https://codepen.io/cahdeemer/pen/ExmeXgp) example, linked from [this "Awesome Container Queries" list](https://github.com/sturobson/Awesome-Container-Queries)..

but alas, that one is even older, being last updated in November 2021.. and the Codepen page doesn't _appear_ to function properly in the same environment as described above.

**Comment by ZaLiTHkA** [comment]
thanks @jzombie for prompting me to try this again.

seems I'm not getting the timeout issue again now.. same computer, no new commits to the repo, same NodeJS version, but Google Chrome (stable, not dev) _literally just_ updated to `103.0.5060.66` as I went to check the current version.

one small difference to the screenshot posted above though, my test page shows `[object Object]` for a few tests, and `true` for all the rest:

![image](https://user-images.githubusercontent.com/2149312/176136457-0a55fadf-7f38-409c-ab9b-277c8fc81eb1.png)

I'm honestly not sure what caused an issue last time I tried, maybe it was just one of those days where nothing "Just Worked"..? regardless, it seems today is better. 😁 

my next step will be to try using the polyfill, but I'm a little busy with _"work"_ work at the moment, so I'll need to dig into this when I get home later.
